### ISSUE TITLE
InternalError when using Character.getName or Character.codePointOf

### ISSUE THREAD

**Issue by bric3** [oracle/graal#3133]
**Describe the issue**
Using either method `Character.getName` or `Character.codePointOf` result in `InternalError`, this is because both of this methods rely on the `CharacterName` class that is initialized from the file `uniNames.dat`. This can be worked around by passing `-H:IncludeResources='java/lang/uniName.dat$'`, but it shouldn't it be included by default ?

```code
Character.codePointOf("LEFT WHITE LENTICULAR BRACKET")
```

```java
private CharacterName()  {
    try (DataInputStream dis = new DataInputStream(new InflaterInputStream(
        AccessController.doPrivileged(new PrivilegedAction<>() {
            public InputStream run() {
                return getClass().getResourceAsStream("uniName.dat");
            }
        })))) {
```

**Steps to reproduce the issue**

```shell
$ echo "public class CharacterReproducer {
  public static void main(String... args) { Character.codePointOf(\"COMBINING HOMOTHETIC ABOVE\"); }
}" > CharacterReproducer.java
$ javac CharacterReproducer.java
$ native-image --native-image-info --verbose --no-fallback -cp . CharacterReproducer CharacterReproducer
Executing [
/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/bin/java \
-XX:+UseParallelGC \
-XX:+UnlockExperimentalVMOptions \
-XX:+EnableJVMCI \
-Dtruffle.TrustAllTruffleRuntimeProviders=true \
-Dtruffle.TruffleRuntime=com.oracle.truffle.api.impl.DefaultTruffleRuntime \
-Dgraalvm.ForcePolyglotInvalid=true \
-Dgraalvm.locatorDisabled=true \
-Dsubstratevm.IgnoreGraalVersionCheck=true \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.aarch64=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.amd64=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.site=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.stack=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.common=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.aarch64=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.amd64=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.sparc=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.meta=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.runtime=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=ALL-UNNAMED \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.sparc=ALL-UNNAMED \
--add-exports=org.graalvm.truffle/com.oracle.truffle.api=ALL-UNNAMED \
--add-opens=jdk.internal.vm.compiler/org.graalvm.compiler.debug=ALL-UNNAMED \
--add-opens=jdk.internal.vm.compiler/org.graalvm.compiler.nodes=ALL-UNNAMED \
--add-opens=jdk.unsupported/sun.reflect=ALL-UNNAMED \
--add-opens=java.base/jdk.internal.module=ALL-UNNAMED \
--add-opens=java.base/jdk.internal.ref=ALL-UNNAMED \
--add-opens=java.base/jdk.internal.reflect=ALL-UNNAMED \
--add-opens=java.base/java.io=ALL-UNNAMED \
--add-opens=java.base/java.lang=ALL-UNNAMED \
--add-opens=java.base/java.lang.reflect=ALL-UNNAMED \
--add-opens=java.base/java.lang.invoke=ALL-UNNAMED \
--add-opens=java.base/java.lang.ref=ALL-UNNAMED \
--add-opens=java.base/java.net=ALL-UNNAMED \
--add-opens=java.base/java.nio=ALL-UNNAMED \
--add-opens=java.base/java.nio.file=ALL-UNNAMED \
--add-opens=java.base/java.security=ALL-UNNAMED \
--add-opens=java.base/javax.crypto=ALL-UNNAMED \
--add-opens=java.base/java.util=ALL-UNNAMED \
--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED \
--add-opens=java.base/sun.security.x509=ALL-UNNAMED \
--add-opens=java.base/jdk.internal.logger=ALL-UNNAMED \
--add-opens=java.desktop/sun.java2d.cmm.profiles=ALL-UNNAMED \
--add-opens=org.graalvm.sdk/org.graalvm.nativeimage.impl=ALL-UNNAMED \
--add-opens=org.graalvm.sdk/org.graalvm.polyglot=ALL-UNNAMED \
--add-opens=org.graalvm.truffle/com.oracle.truffle.polyglot=ALL-UNNAMED \
--add-opens=org.graalvm.truffle/com.oracle.truffle.api.impl=ALL-UNNAMED \
-XX:+UseJVMCINativeLibrary \
-Xss10m \
-Xms1g \
-Xmx13743895344 \
-Duser.country=US \
-Duser.language=en \
-Djava.awt.headless=true \
-Dorg.graalvm.version=20.3.0 \
-Dorg.graalvm.config= \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Xshare:off \
--module-path \
/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/truffle/truffle-api.jar \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-javaagent:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/svm.jar \
-cp \
/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/llvm-wrapper-shadowed.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/svm.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/objectfile.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/llvm-platform-specific-shadowed.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/javacpp-shadowed.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/svm-llvm.jar:/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/builder/pointsto.jar \
'com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus' \
-imagecp \
/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/library-support.jar:/Users/bric3/opensource/jufmt \
-H:Path=/Users/bric3/opensource/jufmt \
-H:+DumpTargetInfo \
-H:FallbackThreshold=0 \
-H:CLibraryPath=/Users/bric3/.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/clibraries/darwin-amd64 \
-H:Class=CharacterReproducer \
-H:Name=character-reproducer \

]
[character-reproducer:66428]    classlist:     945.69 ms,  0.96 GB
[character-reproducer:66428]        (cap):   2,972.22 ms,  0.96 GB
[character-reproducer:66428]        setup:   4,135.83 ms,  0.96 GB
# Building image for target platform: org.graalvm.nativeimage.Platform$DARWIN_AMD64
# Using native toolchain:
#   Name: LLVM (clang)
#   Vendor: apple
#   Version: 12.0.0
#   Target architecture: x86_64
#   Path: /usr/bin/cc
# Using CLibrary: com.oracle.svm.core.c.libc.NoLibC
[character-reproducer:66428]     (clinit):     204.59 ms,  1.22 GB
# Static libraries:
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/clibraries/darwin-amd64/liblibchelper.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/static/darwin-amd64/libnet.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/clibraries/darwin-amd64/libffi.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/clibraries/darwin-amd64/libdarwin.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/static/darwin-amd64/libnio.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/static/darwin-amd64/libjava.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/static/darwin-amd64/libfdlibm.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/static/darwin-amd64/libzip.a
#   ../../.asdf/installs/java/graalvm-20.3.0+java11/lib/svm/clibraries/darwin-amd64/libjvm.a
# Other libraries: -framework Foundation,pthread,dl,z
[character-reproducer:66428]   (typeflow):   3,944.51 ms,  1.22 GB
[character-reproducer:66428]    (objects):   3,339.61 ms,  1.22 GB
[character-reproducer:66428]   (features):     199.92 ms,  1.22 GB
[character-reproducer:66428]     analysis:   7,912.60 ms,  1.22 GB
[character-reproducer:66428]     universe:     330.37 ms,  1.22 GB
[character-reproducer:66428]      (parse):     839.56 ms,  1.22 GB
[character-reproducer:66428]     (inline):   1,104.51 ms,  1.67 GB
[character-reproducer:66428]    (compile):   6,341.83 ms,  2.26 GB
[character-reproducer:66428]      compile:   8,744.51 ms,  2.26 GB
[character-reproducer:66428]        image:   1,046.27 ms,  2.26 GB
[character-reproducer:66428]        write:     281.36 ms,  2.26 GB
[character-reproducer:66428]      [total]:  23,535.75 ms,  2.26 GB

$ ./character-reproducer
Exception in thread "main" java.lang.InternalError
        at java.lang.CharacterName.<init>(CharacterName.java:110)
        at java.lang.CharacterName.getInstance(CharacterName.java:137)
        at java.lang.Character.codePointOf(Character.java:10699)
        at CharacterReproducer.main(CharacterReproducer.java:2)
Caused by: java.lang.NullPointerException
        at java.util.zip.InflaterInputStream.<init>(InflaterInputStream.java:84)
        at java.util.zip.InflaterInputStream.<init>(InflaterInputStream.java:99)
        at java.util.zip.InflaterInputStream.<init>(InflaterInputStream.java:109)
        at java.lang.CharacterName.<init>(CharacterName.java:52)
        ... 3 more
```

**Describe GraalVM and your environment:** graalvm-20.3.0+java11
 - GraalVM version (latest snapshot builds can be found [here](https://github.com/graalvm/graalvm-ce-dev-builds/releases)), or commit id if built from source: **CE 20.3**
 - JDK major version: **11**
 - OS: **mocOS Big Sur 11.1**
 - Architecture: **x64**

**Comment by oubidar-Abderrahim** [comment]
Fixed by commit ae6cf4ee0c75d71dfe44e852874fe0fe61242ca7

**Comment by bric3** [comment]
Cool, thanks @oubidar-Abderrahim !
